frontend/src/Components/Account/Login.js

```diff
@@ -24,13 +24,16 @@ export default function Login() {
     axios
       .post("http://localhost:8000/user/login", { ...form })
       .then((resp) => {
+        toast.success(JSON.stringify(resp.data.message));
         localStorage.setItem("access_token", resp.data.data.access_token);
         navigate("/home");
         setLoading(false);
         window.location.reload();
       })
-      .catch((err) => {
-        setFormError(true);
+      .catch(({ response }) => {
+        if (response) {
+          toast.error("Status " + response.status + " (" + response.statusText + "): " + JSON.stringify(response.data.detail));
```

Comment:
Same as in register.js

Reply:
Thing is, when you give a blank input to the backend (when the return detail is "Unprocessable Identity", it's very hard to understand it without knowing "Unprocessable Identity")
I can remove the response status, but I guess we can keep the `statusText`?

Reply:
`response.data.detail` specifies clearly the error na? 🤔

Reply:
> `response.data.detail` specifies clearly the error na?

![image](https://user-images.githubusercontent.com/43996118/113563547-7e308f00-9625-11eb-8a0a-b6c63e2bf1c6.png)

Reply:
Done 👍🏼 
(Approve before merging. XD)